docs/using/components/configuration/configuration.rst

```diff
@@ -50,6 +50,17 @@ file type where the information is stored or a number of other sections.
 
 .. jsonschema:: schemas/model.yml
 
+.. jsonschema:: schemas/model_definitions.yml#/definitions/abundances/file
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/abundances/uniform
+
+The ``abundance`` section has a possible ``file`` parameter with ``type`` (currently only ``artis`` is allowed)
+and a ``name`` parameter giving a path to a file containing the abundance information.
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/structure/file
+
+.. jsonschema:: schemas/model_definitions.yml#/definitions/structure/specific
+
```

Comment:
Here again you can make a subheading under "Model" named "Structure" and under it, put the following explanatory lines (L64-L70) first and then these two schemas.

Reply:
Yes, Sure! I will do it so as to maintain uniformity.